dashboard/src/components/widget/notification/che-warning-notification.directive.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2015-2018 Red Hat, Inc.
```

Comment:
2019

Reply:
first should be changed here https://github.com/eclipse/che-parent/blob/master/pom.xml#L84 
to pass validation